qubes-keepass.py

```diff
@@ -595,6 +595,35 @@ def get_secret(self) -> str:
         self.item.load_secret_sync()
         return self.item.get_secret().get_text()
 
+
+    def get_totp(self) -> str:
+        '''
+        Obtain the TOTP token from selected entry
+
+        Parameters:
+            None
+
+        Returns:
+            TOTP token
+
+
+        Current issue with self.item.get_attributes().get('TOTP') does not reload the database.
+        This results in a non renual of the TOTP attribute.
+        '''
+
+        if Config.getboolean('view_totp'):
+            try:
+                import pyotp
+            except ImportError:
+                print(f'[-] To use TOTP the library pyotp is needed')
+                return
+            
+        if "totp" in self.item.get_attributes().get('otp'):
+            otp_value = self.item.get_attributes().get('otp').split("?")[1].split("&")[0].split("=")
```

Comment:
What is the actual format here? Is this always the same? Can you provide a example string?